### PR TITLE
Foreman proxy installation instructions

### PR DESCRIPTION
**Pull request by lzap**

### CODE REVIEW

File: _includes/manuals/1.7/4.3.1_smartproxy_installation.md [hunk 1]
```diff
@@ -7,9 +7,65 @@ can be found on the [**Foreman Architecture**](manuals/{{page.version}}/index.ht
...
 In the special case of a smart proxy managing a windows DHCP server, the host machine must be running Windows and support the **netsh dhcp** utility, it does not need to be the Microsoft DHCP server itself.
 
-#### Packages
-
-RPM and Debian packages are available, see the [Install from Packages](manuals/{{page.version}}/index.html#3.3InstallFromPackages) section for configuration and install the `foreman-proxy` package.
+By default, our installer deploys one Foreman Proxy instance on the same host
+Foreman was installed on. This chapter describes installation of additional
+instance(s).
+
+#### Installer
```
Comment (domcleal) [inline _includes/manuals/1.7/4.3.1_smartproxy_installation.md]: Rather than duplicating this, can we link to or combine it with section 3.2.3?

Comment (lzap) [inline _includes/manuals/1.7/4.3.1_smartproxy_installation.md]: I haven't noticed we have that!

Dropped all my changes except the manual installation paragraph which is now in the 3.2.3.

File: _includes/manuals/1.7/3.2.3_installation_scenarios.md [hunk 2]
```diff
@@ -157,3 +157,14 @@ foreman-installer \
...
 Also ensure here that the dhcp-interface argument is updated for the interface to run DHCP on.
+
+#### Manual installation of Smart Proxy
+
+To integrate Foreman Proxy with existing services, manual configuration needs
+to be done. In that case install the `foreman-proxy` package and proceed with
+editing `/etc/foreman-proxy` configuration files. The service name to restart
+is named `foreman-proxy`.
+
+We don't provide details about manual configuration, so the recommended way is
```
Comment (domcleal) [inline _includes/manuals/1.7/3.2.3_installation_scenarios.md:168]: isn't that the main smart proxy config chapter?  I'm not entirely sure I understand this.. :)

**Comment by lzap** [comment]
Hm I think I need to read our manual once again. Too much coffee.

**Pull request closed without merging**